lib/bootloader_setup.pm

```diff
@@ -540,6 +540,8 @@ sub select_bootmenu_more {
         send_key "ret";
     }
     send_key_until_needlematch($tag, get_var('OFW') ? 'up' : 'down', 10, 3);
+    # Redirect linuxrc logs to console when booting from menu: "boot linux system"
+    push @params, "linuxrc.log=/dev/$serialdev" if get_var('LINUXRC_BOOT');
```

Comment:
I believe we should reuse 2 more parameters as we have here: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/bootloader_setup.pm#L406
Maybe we can simply call `bootmenu_default_params`.

Reply:
It's what I was mentioning, if we use debug options such as linuxrc.debug and linuxrc.core, then we get an [extra step](http://waaa-amazing.suse.cz/tests/4219#step/boot_linuxrc/13) for kexec and then if we just type enter, boot fails in some cases.

Reply:
So I thought we may do it in the scope of a follow-up ticket, as we discussed. This being said I am motivated do it right away.

Reply:
Ah, I missed that point. Could you please create follow up ticket and describe this difference in the behavior?

Reply:
Sure

Reply:
https://progress.opensuse.org/issues/54014